hypex/dataset/backends/abstract.py

```diff
@@ -195,11 +195,19 @@ def _get_column_index(
         raise AbstractMethodError
 
     @abstractmethod
-    def get_column_type(self, column_name: str) -> str:
+    def get_column_type(self, column_name: str) -> Union[DefaultRoleTypes, None]:
```

Comment:
This method does not return objects of DefaultRoleTypes, i.e. int, float or bool, it returnes type name